lib/utilities/aclUtils.js

```diff
@@ -233,4 +234,58 @@ aclUtils.sortHeaderGrants =
         return addACLParams;
     };
 
+/**
+ * convertToXml - Converts the `grantInfo` object (defined in `objectGetACL()`)
+ * to an XML DOM string
+ * @param {object} grantInfo - The `grantInfo` object defined in
+ * `objectGetACL()`
+ * @return {string} xml.join('') - The XML DOM string
+ */
+aclUtils.convertToXml = grantInfo => {
+    const { grants, ownerInfo } = grantInfo;
+    const xml = [];
+
+    xml.push('<?xml version="1.0" encoding="UTF-8" standalone="yes"?>',
+             '<AccessControlPolicy>',
+             '<Owner>',
+             `<ID>${ownerInfo.ID}</ID>`,
+             `<DisplayName>${ownerInfo.displayName}</DisplayName>`,
```

Comment:
In S3/conf/authdata.json I had changed the account name from "Bart" to something like "Bart<Simpson". This change did not throw any error and the special character "<" was escaped accordingly. This is why I figured `displayName` needed escaping. 

[`escapeForXML()`](https://github.com/scality/S3/blob/e09c9d666f958e6bef3eb257f084acaed071ca3d/lib/utilities/escapeForXML.js#L5) will not escape "@", but that seems to be okay for XML as per [W3C specs](https://www.w3.org/TR/xml11/#sec-predefined-ent).